backend/apps/search/tests.py

```
@@ -218,3 +218,121 @@ def test_cache_version_bump(self):
 
         version2 = cache.get("search_api_version", 1)
         self.assertEqual(version2, version1 + 1)
+
+
+class MeilisearchIntegrationTests(TestCase):
+    def setUp(self):
+        from unittest.mock import MagicMock
+        self.factory = RequestFactory()
+        self.view = UnifiedSearchView.as_view()
+
+    @patch("apps.search.tasks.get_meili_index")
```

Ruff (0.15.21)

[error] Undefined name `patch`

(F821)